microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -69,25 +69,25 @@ Mobile devices have much less CPU power than desktops and laptops.  Whenever you
 
    ![CPU throttle.](../media/evaluate-performance-performance-capture-settings.msft.png)
 
-   When testing other pages; if you want to ensure that each page works well on low-end mobile devices, set CPU Throttling to **6x slowdown**.  The demo doesn't work well with 6x slowdown, so it just uses 4x slowdown for instructional purposes.
+   If you want to ensure that pages work well on low-end mobile devices, set CPU Throttling to **6x slowdown**.  The demo doesn't work well with 6x slowdown, so it just uses 4x slowdown for instructional purposes.
```

Comment:
```suggestion
   If you want to ensure that pages work well on low-end mobile devices, set **CPU** to **6x slowdown**.  The demo doesn't work well with 6x slowdown, so it just uses 4x slowdown for instructional purposes.
```
Either write in terms of UI labels, or regular words (ie lowercase):
set **CPU** to **6x slowdown**.
set CPU throttling to **6x slowdown**.